app/routes/home.tsx

```
@@ -0,0 +1,21 @@
+import { ToastContainer } from "react-toastify";
+
+import { WeatherHome } from "~/components/weatherHome/weatherHome";
+
+import type { Route } from "./+types/home";
```

ESLint

[error] 'Route' is defined but never used.

(@typescript-eslint/no-unused-vars)